core/templates/pages/contact-page/contact-page.module.ts

```diff
@@ -26,13 +26,31 @@ import { platformFeatureInitFactory, PlatformFeatureService } from 'services/pla
 import { ContactPageComponent } from './contact-page.component';
 import { ContactPageRootComponent } from './contact-page-root.component';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ToastrModule } from 'ngx-toastr';
+
+// Config for ToastrModule (helps in flashing messages and alerts).
+const toastrConfig = {
+  allowHtml: false,
+  iconClasses: {
+    error: 'toast-error',
+    info: 'toast-info',
+    success: 'toast-success',
+    warning: 'toast-warning'
+  },
+  positionClass: 'toast-bottom-right',
+  messageClass: 'toast-message',
+  progressBar: false,
+  tapToDismiss: true,
+  titleClass: 'toast-title'
+};
```

Comment:
```suggestion
// Config for ToastrModule (helps in flashing messages and alerts).
const toastrConfig = {
  allowHtml: false,
  iconClasses: {
    error: 'toast-error',
    info: 'toast-info',
    success: 'toast-success',
    warning: 'toast-warning'
  },
  positionClass: 'toast-bottom-right',
  messageClass: 'toast-message',
  progressBar: false,
  tapToDismiss: true,
  titleClass: 'toast-title'
};
```

I'm not sure I understand the comment, also why we need to define this separately for every page?

Reply:
Removed while resolving merge conflicts as this page is migrated to angular router in the last PR.

Reply:
ToastrModule didn't support lazy loading, so had to move this to root modules for every page.